index.html

```diff
@@ -8,27 +8,32 @@
   </head>
 <body>
     
-  <header><img src="https://i.pinimg.com/736x/27/a7/21/27a72184a8e1cef49a1aea022221cee7.jpg"></header>
+  <header><img src="https://64.media.tumblr.com/c0934f8e00b7a579940142d3bea496f4/tumblr_p07dnaH2KQ1whqd0no2_1280.png"></header>
```

Comment:
here's a few tips on indentation:
if an element is nested inside another element, you should make a new line and indent it. If an element comes after another element closes, they should be on the same line.

Example:

`<header>
    <img>
</header>`

vs:

`<title>Title</title>
<link>link</link>`

Lemme know if that doesn't make sense.

Also prettier can be configured to auto-indent everything

Reply:
When you have time could you show me how to set up prettier

Reply:
I think if you clone the repo template, prettier will be included. I can help you with that when we have work time today (feel free to remind me)